PeerioTechnologies/peerio-mobile
# Fix file rename popup

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/10405/mobile-folder-rename-triggers-two-dialogs
#### Testing instructions  
1- For both, file and folder rename, user should be able to cancel the dialog.
2- `OK` button should say `Update`
3- `Update` button should be disabled if the input name matches the original name.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
